src/VisualStudio/IntegrationTest/New.IntegrationTests/InProcess/EditorInProcess.cs

```diff
@@ -415,12 +415,33 @@ public async Task SetUseSuggestionModeAsync(bool forDebuggerTextView, bool value
     {
         if (await IsUseSuggestionModeOnAsync(forDebuggerTextView, cancellationToken) != value)
         {
-            await TestServices.Shell.ExecuteCommandAsync(VSConstants.VSStd2KCmdID.ToggleConsumeFirstCompletionMode, cancellationToken);
-            if (await IsUseSuggestionModeOnAsync(forDebuggerTextView, cancellationToken) != value)
+            await UpdateUseSuggestionModeAsync();
+            var useSuggestionMode = await IsUseSuggestionModeOnAsync(forDebuggerTextView, cancellationToken);
```

Comment:
These test started failing to set the option when resetting options between tests.  Talked to the editor - the olecommand version only works with an active text view which we may not have at that point.  

Instead we can just use the editor API to set the option, like we do when we read it.